app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -26,6 +29,7 @@ import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.rule.ActivityTestRule
 import com.google.common.truth.Truth.assertThat
 import dagger.Component
+import kotlinx.android.synthetic.main.ongoing_story_card.view.*
```

Comment:
We shouldn't use ktx synthetic references as they're now deprecated: https://developer.android.com/topic/libraries/view-binding/migration (though we don't use Jetpack view binding since we use AndroidX databinding, instead).

Reply:
It was unused import. Removed it.